spec/system/publishers_can_manage_settings_spec.rb

```diff
@@ -1,15 +1,18 @@
 require "rails_helper"
 
-RSpec.describe "Publishers can manage settings" do
+RSpec.describe "Publishers can manage organization/school profile" do
```

Comment:
Think we tend to use British English spelling 🇬🇧 😂

Reply:
Ah, you mean correct English :P Yeah, my RubyMine keeps suggesting US spellings...